tools/tilt/Tiltfile

```diff
@@ -6,6 +6,8 @@ settings.update(read_json(
     default={},
 ))
 
+docker_prune_settings(max_age_mins=1440)
```

Comment:
This speeds up `make tilt` dramatically, keeps image layers around for 24 hrs instead of rebuilding all from scratch every time.